src/raw_reader.rs

```diff
@@ -1,152 +1,47 @@
 use crate::raw_symbol_token::RawSymbolToken;
-use crate::result::IonResult;
-use crate::types::decimal::Decimal;
-use crate::types::timestamp::Timestamp;
+use crate::stream_reader::StreamReader;
 use crate::types::IonType;
-use bigdecimal::BigDecimal;
-use chrono::{DateTime, FixedOffset};
+use std::fmt::{Display, Formatter};
 
-/**
```

Comment:
PR tour: this trait has been subsumed by the `StreamReader` trait. See just below this large deletion for the new `RawReader` trait, which is a simple alias for `StreamReader<Item=RawStreamItem, Symbol=RawSymbolToken>`.